docs/extending/localtunneldebugger.md

```diff
@@ -0,0 +1,122 @@
+# Adding Providers to the Local Tunnel Debugger
+
+The Local Tunnel Debugger is intended to provide a local debugging option that includes
+the benefits of remote debugging. Traffic from the cluster is rerouted to the version of the service 
```

Comment:
Nit: maybe `application` instead of `services`? Users might not only debug services

Reply:
Sure!